src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const listNode = document.querySelector('ul');
+const [...employees] = listNode.children;
+
+sortList(employees);
+listNode.append(...employees);
+getEmployees(employees);
+
+function sortList(list) {
+  list.sort((a, b) => {
```

Comment:
would be better yo copy `list` here
like
`return [...list].sort...`

and use like

`listNode.append(sortList(employees))`